src/components/transactions/message-view/SubmitProposalMessageDetails.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <div class="tx__content__caption">
+      <p>
+        Submitted {{ transaction.value.proposal_type.toLowerCase() }} proposal
+        <b>{{ coin.amount | toAtoms | prettyLong }}</b>
+        <span>{{ coin.denom | viewDenom }}</span>
+      </p>
+    </div>
+    <div class="tx__content__information">
+      Title
+      <i>{{ transaction.value.title }}</i>
+    </div>
+  </div>
+</template>
+
+<script>
+import { atoms, viewDenom, prettyLong } from "scripts/num.js"
+import { formatBech32 } from "src/filters"
+
+export default {
+  name: `submit-proposal-message-details`,
+  filters: {
+    atoms,
+    viewDenom,
+    prettyLong,
+    formatBech32
+  },
+  props: {
+    transaction: {
+      type: Object,
+      required: true
+    },
+    coin: {
+      type: Object,
+      required: true
+    },
+    validators: {
+      type: Object,
+      required: true
+    }
+  }
+}
+</script>
+
+<style>
+.tx__content__information,
+.tx__content__information > * {
+  display: flex;
```

Comment:
styles are copied a lot. should we refactor them?
also: styles are not scoped. should be scoped I think.

Reply:
Yep. I wrote about this in the comment above. I wanted to get it right before changing this.